modules/distribution-zones/src/main/java/org/apache/ignite/internal/distributionzones/rebalance/RebalanceUtil.java

```diff
@@ -324,15 +324,6 @@ public static CompletableFuture<?>[] triggerAllTablePartitionsRebalance(
     /** Key prefix for switch append assignments. */
     public static final String ASSIGNMENTS_SWITCH_APPEND_PREFIX = "assignments.switch.append.";
```

Comment:
Curious about why we do use APPEND_PREFIX only in RebalanceUtil itself, switchAppendKey with package-private scope but do register watch fo REDUCE_PREFIX in TableManager. That's suspicious. Not related to your PR though.